### PR TITLE
Primary Reason Story -- functionality 

### PR DESCRIPTION
**Pull request by alexsoble**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/34)

## What does this change?

+ This PR includes core functionality for the Primary Issue story. However, this PR does not finish the Story by itself — see below for more on what's not included in this PR. 

## What's still TODO?

+ **_Alert states_**: This PR does not include all the required changes to alert states. Setting the Primary Issue as `required` in the Django back-end automatically enables HTML5 client-side validation on this page, which is currently messy and un-styled. I may end up adding styled client-side and server-side validation for this Story, since adding server-side validation seems to automatically turn on client-side validation. 
+ **_Form width_**: The page currently doesn't use the grid/column system spelled out in the designs, which affects the width of the form and the radio button areas. To match the design spec to a high degree of fidelity, I will need to implement the specified grid system. This may or may not be a blocker for the Primary Issue story — can discuss with designers. 

## Screenshots (for front-end PR):

page2

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

+ This PR touches Sass that applies to mutliple pages — for example, Sass for `legend` and `ul`/`li`. Would you mind double-checking to make sure this doesn't introduce any style regressions in other pages in addition to the usual checks?
+ I had a hard time passing the "choices to examples" data to the front-end through Django's form structure. The best solution I found was to pass the data in as a widget attribute. This has the inelegant side-effect of appending a lengthy `choices_to_examples` data attribute to each `<input>` element... but it does work. If you can think of a more elegant way to do this, that would be wonderful! That said, it's not a functionality issue, more a potential-code-style-improvement.

### CODE REVIEW

**Comment by alexsoble** [comment]
Starting on server-side alert styles and functionality here: #134.

**Comment by LindsayYoung** [comment]
Looking great so far @alexsoble . 

Only issue I found so far is that I can't tab through the radios. 

Also, want to add an a11y check test for this page during pairing today?

**Comment by alexsoble** [comment]
@LindsayYoung For radio buttons, the up and down keys are used to navigation between options instead of the tab key — see the chart under "Keyboard Testing" for more here: 

https://webaim.org/techniques/keyboard/

Would love to look at this together during pairing!

File: pa11y_scripts/.pa11y-local.json [hunk 1]
```diff
@@ -21,12 +21,28 @@
...
+          "actions": [
+            "wait for element #id_0-contact_phone to be visible",
+            "click element #submit-next",
+            "wait for element #id_1-primary_complaint to be visible",
+            "click element #id_1-primary_complaint_0",
+            "click element #submit-next",
+            "wait for element #id_2-protected_class to be visible",
+            "click element #id_2-protected_class_0",
+            "click element #submit-next",
+            "wait for element #3-violation_summary to be visible"
```
Comment (LindsayYoung) [inline pa11y_scripts/.pa11y-local.json]: I figured out my issue, for this line I needed it to be `#id_3-violation_summary` for it to run.

File: pa11y_scripts/.pa11y-ci-p4.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (LindsayYoung) [inline pa11y_scripts/.pa11y-ci-p4.json]: This is great, let's add it to circle here https://github.com/usdoj-crt/crt-portal/blob/a3790991ce73a8cbea7f3b895767321390a8f7ed/.circleci/config.yml#L66

File: pa11y_scripts/.pa11y-local.json (hunk 1 above)
Comment (alexsoble) [reply pa11y_scripts/.pa11y-local.json]: Oooooh, great catch! Thank you @LindsayYoung!

File: pa11y_scripts/.pa11y-ci-p4.json (hunk 2 above)
Comment (alexsoble) [reply pa11y_scripts/.pa11y-ci-p4.json]: Another great catch, thanks!

**Pull request merged**